ui/test-file/summary/index.tsx

```diff
@@ -188,9 +195,8 @@ export default function FileSummary({
             }}
           />
         </Tooltip>
-        {haveSnapshotFailures && (
```

Comment:
Don't want want to show this button only when there is a snapshot failure?

Reply:
Correct. I think the button to generate snapshots should always be available. Most times you know that you have made a change that will break the snapshots. It is a waste of time to have to run the tests to wait for the button to appear and then press it. I would prefer to have the option to generate the snapshots when I know they need generating.

Reply:
That's a good point.